2013/11/25 01:37:02 [error] [exception.CDbException] SQLSTATE[HY000] [2002] No such file or directory
2013/11/25 01:37:02 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection.' in /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php:388
Stack trace:
#0 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CModule.php(387): CDbConnection->init()
#3 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CModule.php(103): CModule->getComponent('db')
#4 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CFormModel.php(39): ModelCode->init()
#6 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(282): CController->run('index')
#13 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /Users/sol/Documents/GoogleAppEngine/stirplateio/webapp/index.php(41): CApplication->run()
#16 /Applications/Dev/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/devappserver2/php/setup.php(103): require('/Users/sol/Docu...')
#17 {main}
REQUEST_URI=/gii/model/index
HTTP_REFERER=http://localhost:9080/gii/default/index
---
2013/11/25 01:37:14 [error] [exception.CDbException] SQLSTATE[HY000] [2002] No such file or directory
2013/11/25 01:37:14 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection.' in /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php:388
Stack trace:
#0 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CModule.php(387): CDbConnection->init()
#3 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CModule.php(103): CModule->getComponent('db')
#4 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CFormModel.php(39): ModelCode->init()
#6 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(282): CController->run('index')
#13 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /Users/sol/Documents/GoogleAppEngine/stirplateio/webapp/index.php(41): CApplication->run()
#16 /Applications/Dev/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/devappserver2/php/setup.php(103): require('/Users/sol/Docu...')
#17 {main}
REQUEST_URI=/gii/model/index
HTTP_REFERER=http://localhost:9080/gii/default/index
---
2013/11/25 01:38:07 [error] [exception.CDbException] SQLSTATE[HY000] [2002] No such file or directory
2013/11/25 01:38:07 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection.' in /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php:388
Stack trace:
#0 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CModule.php(387): CDbConnection->init()
#3 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CModule.php(103): CModule->getComponent('db')
#4 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CFormModel.php(39): ModelCode->init()
#6 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(282): CController->run('index')
#13 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /Users/sol/Documents/GoogleAppEngine/stirplateio/webapp/index.php(41): CApplication->run()
#16 /Applications/Dev/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/devappserver2/php/setup.php(103): require('/Users/sol/Docu...')
#17 {main}
REQUEST_URI=/gii/model/index
HTTP_REFERER=http://localhost:9080/gii/default/index
---
2013/11/25 01:39:03 [error] [exception.CDbException] SQLSTATE[HY000] [2002] No such file or directory
2013/11/25 01:39:03 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection.' in /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php:388
Stack trace:
#0 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CModule.php(387): CDbConnection->init()
#3 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CModule.php(103): CModule->getComponent('db')
#4 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CFormModel.php(39): ModelCode->init()
#6 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(282): CController->run('index')
#13 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /Users/sol/Documents/GoogleAppEngine/stirplateio/webapp/index.php(41): CApplication->run()
#16 /Applications/Dev/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/devappserver2/php/setup.php(103): require('/Users/sol/Docu...')
#17 {main}
REQUEST_URI=/gii/model/index
HTTP_REFERER=http://localhost:9080/gii/default/index
---
2013/11/25 01:41:12 [error] [exception.CDbException] SQLSTATE[HY000] [2002] No such file or directory
2013/11/25 01:41:12 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection.' in /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php:388
Stack trace:
#0 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CModule.php(387): CDbConnection->init()
#3 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CModule.php(103): CModule->getComponent('db')
#4 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CFormModel.php(39): ModelCode->init()
#6 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(282): CController->run('index')
#13 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /Users/sol/Documents/GoogleAppEngine/stirplateio/webapp/index.php(41): CApplication->run()
#16 /Applications/Dev/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/devappserver2/php/setup.php(103): require('/Users/sol/Docu...')
#17 {main}
REQUEST_URI=/gii/model/index
HTTP_REFERER=http://localhost:9080/gii/default/index
---
2013/11/25 01:41:26 [error] [exception.CDbException] SQLSTATE[HY000] [2002] No such file or directory
2013/11/25 01:41:26 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection.' in /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php:388
Stack trace:
#0 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CModule.php(387): CDbConnection->init()
#3 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CModule.php(103): CModule->getComponent('db')
#4 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CFormModel.php(39): ModelCode->init()
#6 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(282): CController->run('index')
#13 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /Users/sol/Documents/GoogleAppEngine/stirplateio/webapp/index.php(41): CApplication->run()
#16 /Applications/Dev/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/devappserver2/php/setup.php(103): require('/Users/sol/Docu...')
#17 {main}
REQUEST_URI=/gii/model/index
HTTP_REFERER=http://localhost:9080/gii/default/index
---
2013/11/25 01:41:27 [error] [exception.CDbException] SQLSTATE[HY000] [2002] No such file or directory
2013/11/25 01:41:27 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection.' in /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php:388
Stack trace:
#0 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CModule.php(387): CDbConnection->init()
#3 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CModule.php(103): CModule->getComponent('db')
#4 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CFormModel.php(39): ModelCode->init()
#6 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(282): CController->run('index')
#13 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /Users/sol/Documents/GoogleAppEngine/stirplateio/webapp/index.php(41): CApplication->run()
#16 /Applications/Dev/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/devappserver2/php/setup.php(103): require('/Users/sol/Docu...')
#17 {main}
REQUEST_URI=/gii/model/index
HTTP_REFERER=http://localhost:9080/gii/default/index
---
2013/11/25 01:44:02 [error] [exception.CDbException] SQLSTATE[HY000] [2002] No such file or directory
2013/11/25 01:44:02 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection.' in /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php:388
Stack trace:
#0 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CModule.php(387): CDbConnection->init()
#3 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CModule.php(103): CModule->getComponent('db')
#4 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CFormModel.php(39): ModelCode->init()
#6 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(282): CController->run('index')
#13 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /Users/sol/Documents/GoogleAppEngine/stirplateio/webapp/index.php(41): CApplication->run()
#16 /Applications/Dev/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/devappserver2/php/setup.php(103): require('/Users/sol/Docu...')
#17 {main}
REQUEST_URI=/gii/model/index
HTTP_REFERER=http://localhost:9080/gii/default/index
---
2013/11/25 01:45:00 [error] [exception.CDbException] SQLSTATE[HY000] [2002] No such file or directory
2013/11/25 01:45:00 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection.' in /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php:388
Stack trace:
#0 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CModule.php(387): CDbConnection->init()
#3 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CModule.php(103): CModule->getComponent('db')
#4 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CFormModel.php(39): ModelCode->init()
#6 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(282): CController->run('index')
#13 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /Users/sol/Documents/GoogleAppEngine/stirplateio/webapp/index.php(41): CApplication->run()
#16 /Applications/Dev/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/devappserver2/php/setup.php(103): require('/Users/sol/Docu...')
#17 {main}
REQUEST_URI=/gii/model/index
HTTP_REFERER=http://localhost:9080/gii/default/index
---
2013/11/25 01:45:03 [error] [exception.CDbException] SQLSTATE[HY000] [2002] No such file or directory
2013/11/25 01:45:03 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection.' in /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php:388
Stack trace:
#0 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CModule.php(387): CDbConnection->init()
#3 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CModule.php(103): CModule->getComponent('db')
#4 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CFormModel.php(39): ModelCode->init()
#6 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(282): CController->run('index')
#13 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /Users/sol/Documents/GoogleAppEngine/stirplateio/webapp/index.php(41): CApplication->run()
#16 /Applications/Dev/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/devappserver2/php/setup.php(103): require('/Users/sol/Docu...')
#17 {main}
REQUEST_URI=/gii/model/index
HTTP_REFERER=http://localhost:9080/gii/default/index
---
2013/11/25 01:50:22 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection.connectionString cannot be empty.' in /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php:370
Stack trace:
#0 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CModule.php(387): CDbConnection->init()
#3 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CModule.php(103): CModule->getComponent('db')
#4 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CFormModel.php(39): ModelCode->init()
#6 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(282): CController->run('index')
#13 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /Users/sol/Documents/GoogleAppEngine/stirplateio/webapp/index.php(41): CApplication->run()
#16 /Applications/Dev/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/devappserver2/php/setup.php(103): require('/Users/sol/Docu...')
#17 {main}
REQUEST_URI=/gii/model/index
HTTP_REFERER=http://localhost:9080/gii/default/index
---
2013/11/25 01:50:29 [error] [exception.CDbException] SQLSTATE[HY000] [2002] No such file or directory
2013/11/25 01:50:29 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection.' in /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php:388
Stack trace:
#0 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CModule.php(387): CDbConnection->init()
#3 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CModule.php(103): CModule->getComponent('db')
#4 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CFormModel.php(39): ModelCode->init()
#6 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(282): CController->run('index')
#13 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /Users/sol/Documents/GoogleAppEngine/stirplateio/webapp/index.php(41): CApplication->run()
#16 /Applications/Dev/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/devappserver2/php/setup.php(103): require('/Users/sol/Docu...')
#17 {main}
REQUEST_URI=/gii/model/index
HTTP_REFERER=http://localhost:9080/gii/default/index
---
2013/11/25 01:50:50 [error] [exception.CDbException] SQLSTATE[HY000] [2002] No such file or directory
2013/11/25 01:50:50 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection.' in /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php:388
Stack trace:
#0 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CModule.php(387): CDbConnection->init()
#3 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CModule.php(103): CModule->getComponent('db')
#4 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CFormModel.php(39): ModelCode->init()
#6 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(282): CController->run('index')
#13 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /Users/sol/Documents/GoogleAppEngine/stirplateio/webapp/index.php(41): CApplication->run()
#16 /Applications/Dev/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/devappserver2/php/setup.php(103): require('/Users/sol/Docu...')
#17 {main}
REQUEST_URI=/gii/model/index
HTTP_REFERER=http://localhost:9080/gii/default/index
---
2013/11/25 01:52:46 [error] [exception.CDbException] SQLSTATE[HY000] [2002] No such file or directory
2013/11/25 01:52:46 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection.' in /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php:388
Stack trace:
#0 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CModule.php(387): CDbConnection->init()
#3 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CModule.php(103): CModule->getComponent('db')
#4 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CFormModel.php(39): ModelCode->init()
#6 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(282): CController->run('index')
#13 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /Users/sol/Documents/GoogleAppEngine/stirplateio/webapp/index.php(41): CApplication->run()
#16 /Applications/Dev/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/devappserver2/php/setup.php(103): require('/Users/sol/Docu...')
#17 {main}
REQUEST_URI=/gii/model/index
HTTP_REFERER=http://localhost:9080/gii/default/index
---
2013/11/25 01:53:50 [error] [exception.CDbException] SQLSTATE[HY000] [2002] No such file or directory
2013/11/25 01:53:50 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection.' in /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php:388
Stack trace:
#0 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CModule.php(387): CDbConnection->init()
#3 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CModule.php(103): CModule->getComponent('db')
#4 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CFormModel.php(39): ModelCode->init()
#6 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(282): CController->run('index')
#13 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /Users/sol/Documents/GoogleAppEngine/stirplateio/webapp/index.php(41): CApplication->run()
#16 /Applications/Dev/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/devappserver2/php/setup.php(103): require('/Users/sol/Docu...')
#17 {main}
REQUEST_URI=/gii/model/index
HTTP_REFERER=http://localhost:9080/gii/default/index
---
2013/11/25 01:56:55 [error] [exception.CDbException] SQLSTATE[HY000] [2002] No such file or directory
2013/11/25 01:56:55 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection.' in /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php:388
Stack trace:
#0 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CModule.php(387): CDbConnection->init()
#3 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CModule.php(103): CModule->getComponent('db')
#4 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/generators/crud/CrudCode.php(44): CModule->__get('db')
#5 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CFormModel.php(39): CrudCode->init()
#6 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(282): CController->run('index')
#13 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(141): CWebApplication->runController('gii/crud/index')
#14 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /Users/sol/Documents/GoogleAppEngine/stirplateio/webapp/index.php(41): CApplication->run()
#16 /Applications/Dev/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/devappserver2/php/setup.php(103): require('/Users/sol/Docu...')
#17 {main}
REQUEST_URI=/gii/crud/index
HTTP_REFERER=http://localhost:9080/gii/controller/index
---
2013/11/25 01:58:32 [error] [exception.CDbException] SQLSTATE[HY000] [2002] No such file or directory
2013/11/25 01:58:32 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection.' in /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php:388
Stack trace:
#0 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CModule.php(387): CDbConnection->init()
#3 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CModule.php(103): CModule->getComponent('db')
#4 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/generators/crud/CrudCode.php(44): CModule->__get('db')
#5 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CFormModel.php(39): CrudCode->init()
#6 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(282): CController->run('index')
#13 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(141): CWebApplication->runController('gii/crud/index')
#14 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /Users/sol/Documents/GoogleAppEngine/stirplateio/webapp/index.php(41): CApplication->run()
#16 /Applications/Dev/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/devappserver2/php/setup.php(103): require('/Users/sol/Docu...')
#17 {main}
REQUEST_URI=/gii/crud/index
HTTP_REFERER=http://localhost:9080/gii/controller/index
---
2013/11/25 01:58:44 [error] [exception.CDbException] SQLSTATE[HY000] [2002] Connection refused
2013/11/25 01:58:44 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection.' in /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php:388
Stack trace:
#0 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CModule.php(387): CDbConnection->init()
#3 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CModule.php(103): CModule->getComponent('db')
#4 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/generators/crud/CrudCode.php(44): CModule->__get('db')
#5 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CFormModel.php(39): CrudCode->init()
#6 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(282): CController->run('index')
#13 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(141): CWebApplication->runController('gii/crud/index')
#14 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /Users/sol/Documents/GoogleAppEngine/stirplateio/webapp/index.php(41): CApplication->run()
#16 /Applications/Dev/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/devappserver2/php/setup.php(103): require('/Users/sol/Docu...')
#17 {main}
REQUEST_URI=/gii/crud/index
HTTP_REFERER=http://localhost:9080/gii/controller/index
---
2013/11/25 02:04:43 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "login".' in /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php:286
Stack trace:
#0 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(141): CWebApplication->runController('login')
#1 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /Users/sol/Documents/GoogleAppEngine/stirplateio/webapp/index.php(41): CApplication->run()
#3 /Applications/Dev/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/devappserver2/php/setup.php(103): require('/Users/sol/Docu...')
#4 {main}
REQUEST_URI=/login
---
2013/11/25 02:05:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "demo.omnisci.org".' in /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php:286
Stack trace:
#0 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(141): CWebApplication->runController('demo.omnisci.or...')
#1 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /Users/sol/Documents/GoogleAppEngine/stirplateio/webapp/index.php(41): CApplication->run()
#3 /Applications/Dev/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/devappserver2/php/setup.php(103): require('/Users/sol/Docu...')
#4 {main}
REQUEST_URI=/demo.omnisci.org
HTTP_REFERER=http://localhost:9080/
---
2013/11/25 02:06:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "demo.omnisci.org".' in /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php:286
Stack trace:
#0 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/web/CWebApplication.php(141): CWebApplication->runController('demo.omnisci.or...')
#1 /Users/sol/Documents/GoogleAppEngine/stirplateio/yii-framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /Users/sol/Documents/GoogleAppEngine/stirplateio/webapp/index.php(41): CApplication->run()
#3 /Applications/Dev/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/devappserver2/php/setup.php(103): require('/Users/sol/Docu...')
#4 {main}
REQUEST_URI=/demo.omnisci.org
HTTP_REFERER=http://localhost:9080/
---
2013/12/02 16:47:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "study".' in /Users/mine/Desktop/source_code/yii-framework/web/CWebApplication.php:286
Stack trace:
#0 /Users/mine/Desktop/source_code/yii-framework/web/CWebApplication.php(141): CWebApplication->runController('study')
#1 /Users/mine/Desktop/source_code/yii-framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /Users/mine/Desktop/source_code/webapp/index.php(41): CApplication->run()
#3 /Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/devappserver2/php/setup.php(103): require('/Users/mine/Des...')
#4 {main}
REQUEST_URI=/study
---
2013/12/02 16:47:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "study/list".' in /Users/mine/Desktop/source_code/yii-framework/web/CWebApplication.php:286
Stack trace:
#0 /Users/mine/Desktop/source_code/yii-framework/web/CWebApplication.php(141): CWebApplication->runController('study/list')
#1 /Users/mine/Desktop/source_code/yii-framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /Users/mine/Desktop/source_code/webapp/index.php(41): CApplication->run()
#3 /Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/devappserver2/php/setup.php(103): require('/Users/mine/Des...')
#4 {main}
REQUEST_URI=/study/list
---
2013/12/02 17:26:44 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "index.php".' in /Users/mine/Desktop/source_code/yii-framework/web/CWebApplication.php:286
Stack trace:
#0 /Users/mine/Desktop/source_code/yii-framework/web/CWebApplication.php(141): CWebApplication->runController('index.php')
#1 /Users/mine/Desktop/source_code/yii-framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /Users/mine/Desktop/source_code/webapp/index.php(41): CApplication->run()
#3 /Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/devappserver2/php/setup.php(103): require('/Users/mine/Des...')
#4 {main}
REQUEST_URI=/index.php
---
2013/12/02 17:28:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "index.php".' in /Users/mine/Desktop/source_code/yii-framework/web/CWebApplication.php:286
Stack trace:
#0 /Users/mine/Desktop/source_code/yii-framework/web/CWebApplication.php(141): CWebApplication->runController('index.php')
#1 /Users/mine/Desktop/source_code/yii-framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /Users/mine/Desktop/source_code/webapp/index.php(41): CApplication->run()
#3 /Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/devappserver2/php/setup.php(103): require('/Users/mine/Des...')
#4 {main}
REQUEST_URI=/index.php
---
2013/12/02 17:28:47 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "index.php".' in /Users/mine/Desktop/source_code/yii-framework/web/CWebApplication.php:286
Stack trace:
#0 /Users/mine/Desktop/source_code/yii-framework/web/CWebApplication.php(141): CWebApplication->runController('index.php')
#1 /Users/mine/Desktop/source_code/yii-framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /Users/mine/Desktop/source_code/webapp/index.php(41): CApplication->run()
#3 /Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/devappserver2/php/setup.php(103): require('/Users/mine/Des...')
#4 {main}
REQUEST_URI=/index.php
---
2013/12/02 17:28:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "index-test.php".' in /Users/mine/Desktop/source_code/yii-framework/web/CWebApplication.php:286
Stack trace:
#0 /Users/mine/Desktop/source_code/yii-framework/web/CWebApplication.php(141): CWebApplication->runController('index-test.php')
#1 /Users/mine/Desktop/source_code/yii-framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /Users/mine/Desktop/source_code/webapp/index.php(41): CApplication->run()
#3 /Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/devappserver2/php/setup.php(103): require('/Users/mine/Des...')
#4 {main}
REQUEST_URI=/index-test.php
---
2013/12/02 17:29:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "index-test.php".' in /Users/mine/Desktop/source_code/yii-framework/web/CWebApplication.php:286
Stack trace:
#0 /Users/mine/Desktop/source_code/yii-framework/web/CWebApplication.php(141): CWebApplication->runController('index-test.php')
#1 /Users/mine/Desktop/source_code/yii-framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /Users/mine/Desktop/source_code/webapp/index.php(41): CApplication->run()
#3 /Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/devappserver2/php/setup.php(103): require('/Users/mine/Des...')
#4 {main}
REQUEST_URI=/index-test.php
---
2013/12/02 17:29:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "index-test.php".' in /Users/mine/Desktop/source_code/yii-framework/web/CWebApplication.php:286
Stack trace:
#0 /Users/mine/Desktop/source_code/yii-framework/web/CWebApplication.php(141): CWebApplication->runController('index-test.php')
#1 /Users/mine/Desktop/source_code/yii-framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /Users/mine/Desktop/source_code/webapp/index.php(41): CApplication->run()
#3 /Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/devappserver2/php/setup.php(103): require('/Users/mine/Des...')
#4 {main}
REQUEST_URI=/index-test.php
---
2013/12/02 17:29:46 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "study/new".' in /Users/mine/Desktop/source_code/yii-framework/web/CWebApplication.php:286
Stack trace:
#0 /Users/mine/Desktop/source_code/yii-framework/web/CWebApplication.php(141): CWebApplication->runController('study/new')
#1 /Users/mine/Desktop/source_code/yii-framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /Users/mine/Desktop/source_code/webapp/index.php(41): CApplication->run()
#3 /Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/devappserver2/php/setup.php(103): require('/Users/mine/Des...')
#4 {main}
REQUEST_URI=/study/new
---
